dfusion_rust_core/src/models/mod.rs

```diff
@@ -63,5 +63,5 @@ fn iter_hash<T: Serializable>(item: &T, prev_hash: &H256) -> H256 {
     hasher.input(item.bytes());
     let result = hasher.result();
     let b: Vec<u8> = result.to_vec();
-    H256::from(b.as_slice())
+    H256::from_slice(b.as_slice())
```

Comment:
how did you manage to catch all these? Were the cargo warnings of any help?

Reply:
No, compiler errors. The `From<&[u8]>` implementation was removed (probably because it could panic).